.github/workflows/ubuntu_build.yml

```diff
@@ -55,32 +53,14 @@ jobs:
           source: '.'
           extensions: 'hpp,h,cpp,c,cc'
 
-      - name: checkout dpdk repo
-        uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
-        with:
-          repository: 'DPDK/dpdk'
-          ref: v${{  env.DPDK_VERSION  }}
-          path: dpdk
-
-      - name: Install the build dependency
-        run: |
-          apt-get update -y
-          apt-get install -y sudo git gcc meson python3 python3-pyelftools pkg-config libnuma-dev libjson-c-dev libpcap-dev libgtest-dev libsdl2-dev libsdl2-ttf-dev libssl-dev systemtap-sdt-dev llvm clang
-          apt-get install -y doxygen
-          apt-get install -y make m4 clang llvm zlib1g-dev libelf-dev libcap-ng-dev libcap2-bin gcc-multilib
-
-      - name: Build DPDK from source
-        working-directory: dpdk
-        run: |
-            meson build
-            ninja -C build
-            cd build
-            sudo ninja install
-      - name: Git config
-        working-directory: "${{ github.workspace }}"
+      - name: Install the build dependencies
+        env:
+          BUILD_AND_INSTALL_DPDK: 1
+          BUILD_E810_DRIVER: 1
+          # ICE driver installation is disabled (not supported in GitHub Actions)
+          BUILD_AND_INSTALL_ICE_DRIVER: 0
```

Comment:
This i plan to trigger via path_filters.yml 
we can set them dynamically so this job could replace 
build ffmpeg 
build eBPF XDP
and all others